### DISCUSSION TITLE
Recover account

### DISCUSSION THREAD

**Discussion by olsanchezc** [community/community#30975] [Other Feature Feedback, Questions, & Ideas]
I had double authentication through the phone but my phone was stolen and now I can't authenticate to access the account.
How can I recover the account?

**Comment by radar-blip** [comment]
Assuming you're talking about your github account, you could always try GitHub support here: https://support.github.com/request
Maybe the Account and billing section.

**Comment by airtower-luna** [comment]
See [Recover an account with 2FA](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials) for the options you have. I hope you have your recovery codes, because that's the only easy way.

**Reply by olsanchezc** [reply]
Thank, but I already tried it and any option works for me.
